src/Symfony/Component/Serializer/Normalizer/DenormalizerInterface.php

```diff
@@ -34,7 +34,7 @@ interface DenormalizerInterface
      * @param string $format  Format the given data was extracted from
      * @param array  $context Options available to the denormalizer
      *
-     * @return object
+     * @return object|string
```

Comment:
Can we do that ? is it needed ?

Reply:
I don't know if it is **needed**, but I was thinking that if the function will return at some point a string (when the DateTimeNormalizer fails to instanciate a DateTime object and that the flag datetime_allow_invalid is set to true), we normaly have to signal it in the PHPDoc no ?